generate_iso_metadata.py

```diff
@@ -57,7 +59,9 @@ def generate_iso_metadata(productType, logFile, excelFile, demFile, isoBase):
     description='Generate ISO metadata file from Excel spreadsheet',
     formatter_class=RawTextHelpFormatter)
   parser.add_argument('productType', help='name of the product type')
-  parser.add_argument('logFile', help='name of processing log file')
+  parser.add_argument('dataSource', help='name of the data source')
```

Comment:
What is a "data source"? Would it be possible to add more descriptive text as to what you're reffing to?

Reply:
Currently, I have hooked it up for 'SENTINEL'. I plan to extend this for the legacy data someday.